src/zbeacon.c

```diff
@@ -146,6 +148,11 @@ s_self_prepare_udp (self_t *self)
                     &send_to);
             assert (rc == 0);
         }
+        if (enable_multicast) {
```

Comment:
This doesn't look right - ipv6 will always be disabled regardless of the settings, as it will be overwritten by either v4 multicast or v4 broadcast

Reply:
iow: there's an "else" missing